dist.ini

```diff
@@ -1,68 +1,13 @@
 name    = Finance-MIFIR-CONCAT
-author  = binary.com <BINARY@cpan.org>
+author  = DERIV <DERIV@cpan.org>
```

Comment:
```suggestion
author  = binary.com <BINARY@cpan.org>
```

Reply:
DERIV is right, because it is the first time to release to pause.cpan.org. In such case we will ues deriv account.